code/modules/admin/verbs/randomverbs.dm

```diff
@@ -33,7 +33,7 @@
 	if(usr)
 		if (usr.client)
 			if(usr.client.holder)
-				M << "\bold You hear a voice in your head... \italic [msg]"
+				M << "\bold You hear [ticker.Bible_deity_name ? "the voice of " + ticker.Bible_deity_name : "a voice"] in your head... \italic [msg]"

```

Comment:
Please do, \red and \blue are fairly common, and hated.
\bold and \italics are rare as fuck, and should be hated more so.